Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -171,6 +205,32 @@ public async Task<ParticipantExtendedViewModel> PostParticipantAsync(string user
             return participant;
         }
 
+        // change to ParticipantUploadViewModel: a new viewmodel only for adds custom participants
+        public async Task<ParticipantExtendedViewModel> PostCustomParticipantAsync(string username, CustomParticipantViewModel newCustomParticipant)
+        {
+            var newUsername = GetCustomUnqiueUsername(username);
+
+            await _context.Participant.AddAsync(new Participant
+            {
+                Username = newUsername,
+                SpecifiedGender = newCustomParticipant.SpecifiedGender ?? "U",
```

Comment:
```suggestion
                SpecifiedGender = newCustomParticipant.SpecifiedGender ?? "U",
```
why would this ever be null?